build.sbt

```diff
@@ -238,9 +238,11 @@ lazy val `sbt-shared` = project
   .settings(
     plugin,
     utest,
-//  addSbtPlugin("com.dwijnand" % "sbt-compat" % "1.2.0+2-f30b82f4")
+// once sbt 0.13.17 is released the following line can be used
```

Comment:
sbt 0.13.17 was released: https://github.com/sbt/sbt/releases/tag/v0.13.17

Reply:
doesn't matter, the reason isn't because of sbt 0.13.17, it's because sbt-compat should only be added here for Scala 2.10 and 2.12

Reply:
see https://github.com/coursier/coursier/pull/731#issuecomment-356344602

Reply:
Ok, so we will never be able to use the short syntax?

In that case, we should remove the commented out version completely.